### PR TITLE
Fix npm audit errors

### PR DESCRIPTION
**Pull request by diracdeltas**
Fix https://github.com/brave/web-discovery-project/issues/85

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -45,6 +45,8 @@
   ],
   "homepage": "https://github.com/brave/web-discovery-project#readme",
   "resolutions": {
+    "ansi-regex": "5.0.1",
+    "ansi-html": "mahdyar/ansi-html-community#v0.0.8",
```
Comment (diracdeltas) [inline package.json:49]: ansi-html is no longer maintained so this repo does a one-line fix for it https://github.com/Tjatse/ansi-html/compare/master...mahdyar:master

**Review by remusao — APPROVED** [review]
Thanks for the fix. Is it expected that the package lock diff is so big?

**Comment by diracdeltas** [comment]
> Thanks for the fix. Is it expected that the package lock diff is so big?

yeah, i did a fresh install from package.json to fix some of the audit errors. but i actually don't think that's necessary so i can make it smaller if you want.

**Comment by remusao** [comment]
> > Thanks for the fix. Is it expected that the package lock diff is so big?
> 
> yeah, i did a fresh install from package.json to fix some of the audit errors. but i actually don't think that's necessary so i can make it smaller if you want.

No worries, it should not cause any issue. I just wanted to double-check with you.

**Pull request merged**